FileSpecs/Maps.md

```diff
@@ -45,16 +45,16 @@ The files consist of the following sections:
-| 0x01 |      | A: not with `02`,`04`, `20`, not in Labyrinths        | Indoors?            | Sir Marillon's Tomb, Family home                                                    |
-| 0x02 |      | A: not with `01`,`04`                                 | Outdoors?           | Overworld, Twinlake Graveyard, Twinlake city                                        |
-| 0x04 |      | A: not with `01`, `02`, `20`, `40`                    | Dungeon / no light? | Lord Drebin's Cellar, Twinlake Sewers                                               |
-| 0x08 |      |                                                       |                     | Most maps.  NOT some advanced dungeons, such as Castle of Manyeye (minus the Tower) |
-| 0x10 |      |                                                       |                     | Overworld, Twinlake sewers, Family home. NOT Twinlake city or graveyard.            |
-| 0x20 |      | B: not with `01`, `04`, `40`, `80`, not in Labyrinths | Lyramion Overworld  | All overworld maps                                                                  |
-| 0x40 |      | B: not with `04`, `20`, `40`                          |                     | Twinlake Graveyard, Twinlake City                                                   |
-| 0x80 |      | B: not with `20`, `40`                                |                     | Sir Marillon's Tomb, Twinlake Sewers                                                |
+| Flag | Name | Exclusivity                                           | Meaning            | Example maps                                                                        |
+|------|------|-------------------------------------------------------|--------------------|-------------------------------------------------------------------------------------|
+| 0x01 |      | A: not with `02`,`04`, `20`, not in Labyrinths        | Indoors?           | Sir Marillon's Tomb, Family home                                                    |
+| 0x02 |      | A: not with `01`,`04`                                 | Outdoors?          | Overworld, Twinlake Graveyard, Twinlake city                                        |
+| 0x04 |      | A: not with `01`, `02`, `20`, `40`                    | No light           | Lord Drebin's Cellar, Twinlake Sewers                                               |
+| 0x08 |      |                                                       |                    | Most maps.  NOT some advanced dungeons, such as Castle of Manyeye (minus the Tower) |
+| 0x10 |      |                                                       | Can rest here      | Overworld, Twinlake sewers, Family home. NOT Twinlake city or graveyard.            |
+| 0x20 |      | B: not with `01`, `04`, `40`, `80`, not in Labyrinths | Lyramion Overworld | All overworld maps                                                                  |
```

Comment:
Maybe the ability to use transportations like the eagle.

In Ambermoon there is a flag which switches from indoor 2D player sprite to the smaller world surface player sprite. Technically on the world map there was a set of graphics which includes the walking player but also the player on horseback, eagle, ship, etc.

Maybe it is the same here.

Reply:
Very likely, given that the Ambermoon tilesets don't include eagles / horses / rafts indoors at all, if I remember correctly.  So in practice this most likely only affects the ability to call the eagles (?)